pages/executive/[proposal-id].tsx

```diff
@@ -281,28 +270,29 @@ const ProposalView = ({ proposal }: Props): JSX.Element => {
                       <Text color="onSecondary">
                         {supporter.percent}% ({new BigNumber(supporter.deposits).toFormat(2)} MKR)
                       </Text>
-                      <ExternalLink
-                        href={
-                          supporter.name && supporter.name !== 'Shadow Delegate'
-                            ? `/delegates/${supporter.address}`
-                            : getEtherscanLink(getNetwork(), supporter.address, 'address')
-                        }
-                        target="_blank"
+
+                      <Link
+                        href={{
+                          pathname: `/address/${supporter.address}`,
+                          query: { network }
+                        }}
                       >
-                        {supporter.name ? (
-                          <Text
-                            sx={{ color: 'accentBlue', fontSize: 3, ':hover': { color: 'blueLinkHover' } }}
-                          >
-                            {limitString(supporter.name, 28, '...')}
-                          </Text>
-                        ) : (
-                          <Text
-                            sx={{ color: 'accentBlue', fontSize: 3, ':hover': { color: 'blueLinkHover' } }}
-                          >
-                            {cutMiddle(supporter.address)}
-                          </Text>
-                        )}
-                      </ExternalLink>
+                        <ThemeUILink sx={{ mt: 'auto' }} title="Profile details">
+                          {supporter.name ? (
+                            <Text
+                              sx={{ color: 'accentBlue', fontSize: 3, ':hover': { color: 'blueLinkHover' } }}
```

Comment:
For some reason with this, on hover, the cursor is a text cursor and it doesn't show the URL down on the bottom of the page. I would expect the same behavior as the other links we use around the app (pointer cursor and a URL preview at the bottom of the window).

Reply:
Weird, i'll check it

Reply:
ok fixede